runelite-client/src/main/java/net/runelite/client/plugins/runecraft/RunecraftOverlay.java

```diff
@@ -78,15 +79,12 @@ public Dimension render(Graphics2D graphics)
 			}
 
 			Point location = item.getCanvasLocation();
-			if (location == null)
+			if (location != null)
 			{
-				continue;
+				int value = client.getSetting(varbits);
+				graphics.drawString("" + value, location.getX(), location.getY() + 15);
```

Comment:
Maybe this should be + https://docs.oracle.com/javase/7/docs/api/java/awt/FontMetrics.html#getHeight()?